AMW_angular/io/src/app/deployments/deployments-list.component.html

```diff
@@ -75,7 +75,10 @@
       </tr>
     </thead>
     <tbody>
-      <tr *ngFor="let deployment of deployments" class="deploy-{{ deployment.state }}">
+      <tr
```

Comment:
For me this formatting is harder to read... does you IDE automatically do this?

Reply:
I'm using prettier as code formatter, yes. 
what's the problem exactly:
- that attributes are placed on a new line?
- the closing bracket that is on a new line?

Reply:
That the line is split into multiple lines even though it would fit on one line.